library/lib_window.ks

```diff
@@ -0,0 +1,26 @@
+
+
+@lazyglobal off.
+
+function create_window{
```

Comment:
Yes, but since I'm updating almost every file, all comments, except for the one or two newest are "outdated".